viam-cartographer/src/carto_facade/io.cc

```diff
@@ -0,0 +1,112 @@
+// This is an experimental integration of cartographer into RDK.
```

Comment:
Are the only differences between `io.cc`, `io.h`, `io._test.cc` and `file_handler.cc`, `file_handler.h`, `file_handler_test.cc` (respectively) the new namespaces?

Reply:
That & I've deleted some of the functions & header inclusions which won't be needed in CartoFacade (such as those related to reading PCD data from disk).

Reply:
And the fact that I've renamed variables & functions which used to refer to `map` when they meant `internal_state` to use `internal_state`.